src/Modal/Modal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+import { fromTheme, defaultStyledProps, StyledProps } from '../utils/styled'
+import { Theme } from '../themes'
+
+const ModalWrapper = styled.div`
+  position: absolute;
+  width: 600px;
+`
+
+const Header = styled.div`
+  padding: 15px 30px;
```

Comment:
Please use theme's spacing for this and other paddings/margings. 

```
padding: ${fromTheme(theme => theme.global.spacing)}em ${fromTheme(theme => theme.global.spacing * 2)}em;
```

Helps prevent alignment issues like 
![image](https://user-images.githubusercontent.com/205834/54017887-ab30c000-4187-11e9-9b50-96319088a292.png)